code/modules/jobs/job_types/shaft_miner.dm

```diff
@@ -0,0 +1,70 @@
+/datum/job/mining
+	title = "Shaft Miner"
+	flag = MINER
+	department_head = list("Quartermaster")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 3
+	spawn_positions = 3
+	supervisors = "the quartermaster"
+	selection_color = "#dcba97"
+
+	outfit = /datum/outfit/job/miner
+
+	access = list(ACCESS_MAINT_TUNNELS, ACCESS_MAILSORTING, ACCESS_CARGO, ACCESS_QM, ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MINERAL_STOREROOM)
+	minimal_access = list(ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MAILSORTING, ACCESS_MINERAL_STOREROOM)
+
+	display_order = JOB_DISPLAY_ORDER_SHAFT_MINER
+
+/datum/outfit/job/miner
+	name = "Shaft Miner"
+	jobtype = /datum/job/mining
+
+	belt = /obj/item/pda/shaftminer
+	ears = /obj/item/radio/headset/headset_cargo/mining
+	shoes = /obj/item/clothing/shoes/workboots/mining
+	gloves = /obj/item/clothing/gloves/color/black
+	uniform = /obj/item/clothing/under/rank/miner/lavaland
+	l_pocket = /obj/item/reagent_containers/hypospray/medipen/survival
+	r_pocket = /obj/item/storage/bag/ore	//causes issues if spawned in backpack
+	backpack_contents = list(
+		/obj/item/flashlight/seclite=1,\
+		/obj/item/kitchen/knife/combat/survival=1,\
+		/obj/item/mining_voucher=1,\
+		/obj/item/stack/marker_beacon/ten=1)
+
+	backpack = /obj/item/storage/backpack/explorer
+	satchel = /obj/item/storage/backpack/satchel/explorer
+	duffelbag = /obj/item/storage/backpack/duffelbag
+	box = /obj/item/storage/box/survival_mining
+
+	chameleon_extras = /obj/item/gun/energy/kinetic_accelerator
+
+/datum/outfit/job/miner/equipped
+	name = "Shaft Miner (Equipment)"
+	suit = /obj/item/clothing/suit/hooded/explorer
+	mask = /obj/item/clothing/mask/gas/explorer
+	glasses = /obj/item/clothing/glasses/meson
+	suit_store = /obj/item/tank/internals/oxygen
+	internals_slot = SLOT_S_STORE
+	backpack_contents = list(
```

Comment:
Tbh Now that suit vouchers are a thing you can buy, I dont mind it not being a round start gear thing

Reply:
they should never have been a fucking thing to buy, godamnit Lilly

Reply:
>Never should have been a thing to buy
***No other item is like that***
Legit you can buy ANYTHING that is on a item voucher

Reply:
infinite fire proof suits is not something wanted.